checkov/terraform/graph_builder/local_graph.py

```diff
@@ -295,6 +295,23 @@ def _build_cross_variable_edges(self) -> None:
                     any(self.vertices[e.dest].block_type != BlockType.RESOURCE for e in referenced_vertices):
                 modules = vertex.breadcrumbs.get(CustomAttributes.SOURCE_MODULE, [])
                 self._build_edges_for_vertex(origin_node_index, vertex, aliases, resources_types, True, modules)
+            # if we have an edge of module->provider we need to connect that modules' resources to the provider
+            if vertex.block_type == BlockType.MODULE and \
+                    any(self.vertices[e.dest].block_type == BlockType.PROVIDER for e in referenced_vertices):
+                try:
+                    tf_def: TFDefinitionKey = vertex.config.get(vertex.name, {}).get("__resolved__")
```

Comment:
```suggestion
                    tf_def: TFDefinitionKey = vertex.config.get(vertex.name, {}).get(RESOLVED_MODULE_ENTRY_NAME)
```
please use this const from `checkov/common/util/consts.py`